src/app/core/nav-menu/nav-menu.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import { filter, map, withLatestFrom } from 'rxjs/operators';
+import { Observable, of } from 'rxjs';
+
+import { Route, Subroute } from './models';
+import { appNavRoutes } from './routes';
+import { routerSelectors, State } from '../../root-store';
+
+
+@Injectable()
+export class NavMenuService {
+  constructor(private store: Store<State>) {
+  }
+
+  public getRoutes(): Observable<Route[]> {
+    return of(appNavRoutes);
+  }
+
+  public getSubroutes(): Observable<Subroute[]> {
+    return this.getCurrentRoute().pipe(
+      map(route => route.subroutes)
```

Comment:
Shouldn't you put such selectors in some kind of `index.ts` files? I saw something in other parts.

Reply:
I don't understand you. Can you provide more information?
P.S. This is not a ngrx selector.